### ISSUE TITLE
Китофорин не работает

### ISSUE THREAD

**Issue by flying-george** [TauCetiStation/TauCetiClassic#8536]
#### Подробное описание проблемы
Китофарин не работает вообще
#### Что должно было произойти
Ампутированная рука должна была отрасти
#### Что произошло на самом деле
Рука не отрасла, как было в крови 5 унций, так и осталось
#### Как повторить
Отрубить руку и ввести китофарин
#### Дополнительная информация:
Не знаю, может его фиксили. На вики не написано
Issue reported from  Round ID: 48411 (Tau Ceti Classic (RU))
Testmerges: ```#8511 #8404 #8359 #8343 ```
Reporting client version: 514.1568

**Comment by flying-george** [comment]
Точно, уточнение, играл я за вокса. Может только на них и не работает

**Comment by Rolt146** [comment]
Что бы киф работал, должны соблюдаться условия, а именно - наличие крови. Тоесть должна быть подключена активная капельница, а сам персонаж должен быть сытым, и чем больше к состоянию "бля щас лопну", тем лучше

**Comment by Sakuya-Izayoi** [comment]
> Что бы киф работал, должны соблюдаться условия, а именно - наличие крови. Тоесть должна быть подключена активная капельница, а сам персонаж должен быть сытым, и чем больше к состоянию "бля щас лопну", тем лучше

Ну может быть человек соблюдая все эти условия не получил желаемого результата. @flying-george, уточни.

**Comment by Dominig12** [comment]
В общем я исследовал эту ситуацию.
Во первых, действительно одного укола 5 единиц не хватает, что странно, но при этом при втором уколе когда первые 5 единиц кончаются, рука отрастает практически сразу.

От меня, лекарство херня, в сумме он сожрал на выращивание около 600 нутриентов хумана, так же он был все время на грани смерти из за выжигаемой крови в неадекватном количестве, постоянно прокающих токсинах и асфиксии (при условиях постоянно вливаемой крови, дексалина + и антитокса возможно лечить пациента, при этом постоянно вкалывая нутриенты в огромных количествах) - лекарство не стоит своей стоимости

**Comment by Dominig12** [comment]
Проблема определенно находится в этих двух проках
![image](https://user-images.githubusercontent.com/24685276/151464670-ae48839a-5aee-4778-aef9-8e4c6b4a87d8.png)
Огромный дамаг от первого (что просто неадекватно)
А второй недорегенерирует конечность, потому что буквально не хватает кифоторина, либо времени на регенерацию требуется слишком много (нужно синхронизировать метаболизм кифоторина и прок регенерации, а то кринж)
![image](https://user-images.githubusercontent.com/24685276/151465023-483442d4-00d8-40f0-8671-0d3d89de0fd8.png)

**Comment by Dominig12** [comment]
Кифоторин ужасно прожорлив, чтобы нормализовтаь количество нутриентов у хумана, нужно колоть очень много инжекторов воксов, или жрать нереальное количество еды, что почти невозможно из за неадекватного урона (потеря крови, нутриентов, токсины + асфиксия)

**Comment by Kurshan** [comment]
Когда я кодил кифоторин. Метаболизм был согласован в одном лайф проке. Да и трабл с голодом тогда не было таких как сейчас. Есть давнее подозрение, что его требуется бафнуть для юзабельности.

**Comment by Dominig12** [comment]
> Когда я кодил кифоторин. Метаболизм был согласован в одном лайф проке. Да и трабл с голодом тогда не было таких как сейчас. Есть давнее подозрение, что его требуется бафнуть для юзабельности. А по поводу дамага - так и задумывалось, что его применять только под бикардином и диловеном.

конечно понимаю что бикардин и диловен логично, но сейчас бьет токси и окси уроном, при этом выжигает кровь
Возможно изменили часть кода и вместо бикардина надо жрать другое
